apps/src/templates/export/expo/metro.config.exported_js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  resolver: {
+    assetExts: [
+      "html",
+      "css",
+      "j",
+      "jpg",
+      "jpeg",
+      "png",
+      "gif",
+      "mp3",
+      "woff2",
+      "ttf",
+      "txt"
```

Comment:
Nonblocking: It looks like this list of extensions is the same in metro.config.exported_js and app.json.ejs.  Would it be possible to make them both ejs files and share this list?